apps/services/auth/admin-api/src/app/v2/delegations/delegation-admin.controller.ts

```diff
@@ -30,9 +32,10 @@ import flatMap from 'lodash/flatMap'
 import { isDefined } from '@island.is/shared/utils'
 
 const namespace = '@island.is/auth/delegation-admin'
+const ZENDESK_WEBHOOK_SECRET_GENERAL_MANDATE =
+  process.env.ZENDESK_WEBHOOK_SECRET_GENERAL_MANDATE ?? ''
 
 @UseGuards(IdsUserGuard, ScopesGuard)
-@Scopes(DelegationAdminScopes.read)
```

Comment:
I would prefer to keep the `read` scope on the controller. That way all new method will be at least protected by the scope by default, if not specified explicitly on the method.